images/linux/scripts/installers/swift.sh

```diff
@@ -16,6 +16,7 @@ mv swift-$swift_version-RELEASE-ubuntu$image_label /usr/share/swift
 SWIFT_PATH="/usr/share/swift/usr/bin"
 SWIFT_BIN="$SWIFT_PATH/swift"
 ln -s "$SWIFT_BIN" /usr/local/bin/swift
+ln -s "$SWIFT_BIN" /usr/local/bin/swiftc
```

Comment:
Why we create symlink from `swift` -> `swiftc`? It is the different tools.
- swift is an interactive environment (REPL) for Swift
- swiftc is the Swift compiler

https://stackoverflow.com/questions/57777091/whats-the-difference-between-swift-and-swiftc

Reply:
It's the same binary I guess, the provided link contains comment:

> The compiler isn't delegating anything to a REPL. They're just the same binary. This is very common in the unix world.

Reply:
I see, that makes sense.
Anyway, IMHO, better to make consistent symlinks:
- swift -> swift
- swiftc -> swiftc
Just in case if it changes in future